src/persist/src/indexed/mod.rs

```diff
@@ -75,6 +103,7 @@ struct Pending {
     writes: HashMap<Id, Vec<ColumnarRecords>>,
     responses: Vec<PendingResponse>,
     seals: HashMap<Id, u64>,
+    deleted_trace_batches: Vec<(Id, Vec<TraceBatchMeta>)>,
```

Comment:
sorry if I'm missing something obvious, but where is this Id used?

Reply:
you're not. it wasnt being used. i wasnt sure if it would end up being necessary and forgot to take it out before pushing.